.github/workflows/platformDeploy.yml

```diff
@@ -0,0 +1,270 @@
+name: Build and deploy android, desktop, iOS, and web clients
+
+# This workflow is run when any tag is published
+on:
+  push:
+    tags:
+      - '*'
+  release:
+    types: [created]
+
+env:
+  SHOULD_DEPLOY_PRODUCTION: ${{ github.event_name == 'release' }}
+  DEVELOPER_DIR: /Applications/Xcode_12.3.app/Contents/Developer
```

Comment:
I think `DEVELOPER_DIR` should be passed as an environment variable only to the iOS job

Reply:
Two thoughts: 

1) I think we should match iOS and Desktop on Xcode version
2) I didn't think it hurt to pass to ubuntu builds, but I can make that change, just thought it was more DRY with 1.